views/index.pug

```diff
@@ -10,5 +10,16 @@ block content
 
   for project in projects
     div.well= project.project_name
-      button.btn.btn-default.pull-right(data-toggle='collapse', href='#'+project.id, aria-expanded='false', aria-controls=project.id) view
-      div.collapse(id=project.id) TESTING
+      button.btn.btn-default.pull-right(data-toggle='collapse', href='#'+project.id, aria-expanded='false', aria-controls=project.id)
+       span.glyphicon.glyphicon-triangle-bottom(aria-hidden="true")
+      div.collapse(id=project.id)
+
+       br
```

Comment:
I try to avoid the use of `br`s, instead using padding or margin on block level elements.  [Here's a good SO answer on why](http://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br), but the short story is we should try to use tags where they are semantically meaningful, and a `br` is used to indicate a content break.  Here (and after the `p` tag), it is being used for positioning of elements.

An alternative would be to target the paragraph element directly in CSS, and apply the appropriate amount of margin or padding to that element.  This also has the benefit of flexibility - we need not add styles to specific `br` tags to control spacing, we can apply that spacing style directly to the paragraph element:
```css
.collapse p {
  margin: 15px 0;
}
```
(When only two values are present for margin and padding, the first is used for top and bottom, and the second for left and right.)